zerver/lib/queue.py

```diff
@@ -266,7 +300,7 @@ def setup_tornado_rabbitmq():
 queue_lock = threading.RLock()
 
 def queue_json_publish(queue_name, event, processor):
-    # type: (str, Dict[str, Any], Callable) -> None
+    # type: (str, Dict[str, Any], Callable[[Any], None]) -> None
```

Comment:
I think `event` should be `Mapping[str, Any]` and `processor` should be `Callable[[Mapping[str, Any]], None]`.